tests/rptest/tests/acls_test.py

```diff
@@ -172,66 +191,402 @@ def get_super_client(self):
+        authn_method=None,
+        client_auth=False
+    )  # AttributeError: 'AccessControlListTest' object has no attribute 'admin_user_cert'
+    @ignore(
+        use_tls=False,
+        use_sasl=False,
+        enable_authz=False,
+        authn_method=None,
+        client_auth=False
+    )  # AttributeError: 'AccessControlListTest' object has no attribute 'admin_user_cert'
+    @ignore(
+        use_tls=False,
+        use_sasl=False,
+        enable_authz=None,
+        authn_method=None,
+        client_auth=False
+    )  # AttributeError: 'AccessControlListTest' object has no attribute 'admin_user_cert'
+    @matrix(use_tls=[True, False],
+            use_sasl=[True, False],
+            enable_authz=[True, False, None],
+            authn_method=[None, 'sasl', 'mtls_identity'],
+            client_auth=[True, False])
+    def test_describe_acls(self, use_tls: bool, use_sasl: bool,
+                           enable_authz: Optional[bool],
+                           authn_method: Optional[str], client_auth: bool):
         """
         security::acl_operation::describe, security::default_cluster_name
         """
-        self.prepare_cluster(use_tls, use_sasl, enable_authz, authn_method)
+        self.prepare_cluster(use_tls,
+                             use_sasl,
+                             enable_authz,
+                             authn_method,
+                             client_auth=client_auth)
+
+        def should_pass_w_base_user(use_tls: bool, use_sasl: bool,
+                                    enable_authz: Optional[bool],
+                                    authn_method: Optional[str]):
+            '''
+            These are the config combinations where `rpk acl list` should pass
+            with the base user. All other tests should fail with ClusterAuthzError
+            with the base user. These combinations were pulled from previous
+            iterations of this ducktape test (i.e., AccessControlListTest.test_describe_acls).
+            See https://github.com/redpanda-data/redpanda/blob/38650a7b96707ce4a13d9879d459b08bb0a59bba/tests/rptest/tests/acls_test.py#L217
+            '''
+            if use_tls and not use_sasl and enable_authz == False and authn_method == 'mtls_identity':
+                return True
+            elif use_tls and not use_sasl and enable_authz == None and authn_method == 'mtls_identity':
+                return True
+            elif not use_tls and use_sasl and enable_authz == False and authn_method == 'sasl':
+                return True
+            elif use_tls and use_sasl and enable_authz == False and authn_method == None:
+                return True
+            elif use_tls and use_sasl and enable_authz == False and authn_method == 'mtls_identity':
+                return True
+            elif use_tls and use_sasl and enable_authz == False and authn_method == 'sasl':
+                return True
+            else:
+                return False
```

Comment:
I believe the clearest way to describe the behaviour that's desired is:
```suggestion
            if enable_authz is False:
                return True
            elif authn_method == 'none':
                return True
            elif not use_sasl and enable_authz is not True:
                return True
            else:
                return False
```

Reply:
There are probably cases that should definitely fail above that, though. Maybe `not use_tls and authn_method == "mtls_identity"`